Object[] obj_lst
int64 iden
